test/setTimeoutAt.js

```diff
@@ -4,64 +4,77 @@ const test = require('tape');
 const timers = require('..');
 
 test('setTimeoutAt', function (t) {
-  const originalMaxInterval = timers.maxInterval;
```

Comment:
Sorry about this one. Apparently when I copied the raw file contents from PR #1 (and then added to them), I didn't notice it was using spaces instead of tabs.

Reply:
I noticed last time, but until we add linting rules (I'll do to that later) I wasn't gonna moan about it ;)